Cubical/Data/Int/Properties.agda

```diff
@@ -300,6 +316,102 @@ private
+pos+ zero zero = refl
+pos+ zero (suc n) = pos (zero +ℕ suc n) ≡⟨ +-comm (pos (suc n)) (pos zero) ⟩
+                       pos zero + pos (suc n) ∎
+pos+ (suc m) zero = pos (suc (m +ℕ zero)) ≡⟨ cong pos (cong suc (+-zero m)) ⟩
+                       pos (suc m) + pos zero ∎
+pos+ (suc m) (suc n) = pos (suc m +ℕ suc n)            ≡⟨ cong pos (cong suc (+-suc m n)) ⟩
+                       sucInt (pos (suc (m +ℕ n)))     ≡⟨ cong sucInt (cong sucInt (pos+ m n)) ⟩
+                       sucInt (sucInt (pos m + pos n)) ≡⟨ sucInt+ (pos m) (sucInt (pos n)) ⟩
+                       pos (suc m) + pos (suc n) ∎
+
+negsuc+ : ∀ m n → negsuc (m +ℕ n) ≡ negsuc m - pos n
+negsuc+ zero zero = refl
+negsuc+ zero (suc n) = negsuc (zero +ℕ suc n) ≡⟨ negsuc0+ (negsuc n) ⟩
+                       negsuc zero + negsuc n ≡⟨ cong (negsuc zero +_) (-pos (suc n)) ⟩
+                       negsuc zero - pos (suc n) ∎
+negsuc+ (suc m) zero = negsuc (suc m +ℕ zero) ≡⟨ cong negsuc (cong suc (+-zero m)) ⟩
+                       negsuc (suc m) - pos zero ∎
+negsuc+ (suc m) (suc n) = negsuc (suc m +ℕ suc n)        ≡⟨ cong negsuc (sym (+-suc m (suc n))) ⟩
+                          negsuc (m +ℕ suc (suc n))      ≡⟨ negsuc+ m (suc (suc n)) ⟩
+                          negsuc m - pos (suc (suc n))   ≡⟨ sym (+predInt (negsuc m) (negsuc n)) ⟩
+                          predInt (negsuc m + negsuc n ) ≡⟨ predInt+ (negsuc m) (negsuc n) ⟩
+                          negsuc (suc m) - pos (suc n) ∎
+
+neg+ : ∀ m n → neg (m +ℕ n) ≡ neg m + neg n
+neg+ zero zero = refl
+neg+ zero (suc n) = neg (zero +ℕ suc n) ≡⟨ +-comm (neg (suc n)) (pos zero) ⟩
+                       neg zero + neg (suc n) ∎
+neg+ (suc m) zero = neg (suc (m +ℕ zero)) ≡⟨ cong neg (cong suc (+-zero m)) ⟩
+                       neg (suc m) + neg zero ∎
+neg+ (suc m) (suc n) = neg (suc m +ℕ suc n) ≡⟨ negsuc+ m (suc n) ⟩
+                       neg (suc m) + neg (suc n) ∎
+
+ℕ-anticomm : ∀ m n → m ℕ- n ≡ - (n ℕ- m)
+ℕ-anticomm zero zero = refl
+ℕ-anticomm zero (suc n) = refl
+ℕ-anticomm (suc m) zero = refl
+ℕ-anticomm (suc m) (suc n) = suc m ℕ- suc n ≡⟨ ℕ-anticomm m n ⟩
+                             - (suc n ℕ- suc m) ∎
+
+pos- : ∀ m n → m ℕ- n ≡ pos m - pos n
+pos- zero zero = refl
+pos- zero (suc n) = zero ℕ- suc n ≡⟨ +-comm (negsuc n) (pos zero) ⟩
+                    pos zero - pos (suc n) ∎
+pos- (suc m) zero = refl
+pos- (suc m) (suc n) = suc m ℕ- suc n                       ≡⟨ pos- m n ⟩
+                       pos m - pos n                        ≡⟨ sym (sucPred (pos m - pos n)) ⟩
+                       sucInt (predInt (pos m - pos n))     ≡⟨ cong sucInt (+predInt (pos m) (- pos n)) ⟩
+                       sucInt (pos m + predInt (- (pos n))) ≡⟨ cong sucInt (cong (pos m +_) (predInt-pos n)) ⟩
+                       sucInt (pos m + negsuc n)            ≡⟨ sucInt+negsuc n (pos m) ⟩
+                       pos (suc m) - pos (suc n) ∎
+
+-anticomm : ∀ m n → m - n ≡ - (n - m)
```

Comment:
I would use camelCase, so `-AntiComm` or `-antiComm`. I'm not sure which is best? @ecavallo

Reply:
Same for the other anticomm above

Reply:
I decided to go with `-AntiComm` so it would fit better with the above. Does make it a bit out-of-step with the naming of `+-assoc` and `+-comm`, but that was going to be an issue anyway.